service/telemetry/config.go

```diff
@@ -103,3 +104,11 @@ type MetricsConfig struct {
 	// Address is the [address]:port that metrics exposition should be bound to.
 	Address string `mapstructure:"address"`
 }
+
+// TracesConfig exposes the common Telemetry configuration for collector's internal spans.
+type TracesConfig struct {
+	// Propagators is a comma separated list of TextMapPropagators from the default registered list
+	// (tracecontext, baggage, b3, b3multi, jaeger, xray, or ottrace). The default propagator used in
+	// tracecontext and setting the vale to "none" disables the context propagation altogether.
+	Propagators string `mapstructure:"propagators"`
```

Comment:
Why not a proper list in yaml?

Reply:
Also add a comment that this is experimental, until otel defines standard yaml configuration for the library.

Reply:
Done! updated the PR